### ISSUE TITLE
`cusip._alphabet` is too restrictive: prevents matching valid CUSIPs

### ISSUE THREAD

**Issue by thomkav** [arthurdejong/python-stdnum#337]
The `cusip` module's `is_valid` function returns false for a CUSIP that I would expect would pass, as I found it on a real Fidelity statement (see screenshot of document sample). 

```
cusip.is_valid(`FDIC99425`)
>> False
```

I believe this is because `cusip._alphabet` does not permit "I" or "O":
```
cusip._alphabet
>> '0123456789ABCDEFGH JKLMN PQRSTUVWXYZ*@#'
```

<img width="492" alt="Screen Shot 2022-10-16 at 11 50 45 AM" src="https://user-images.githubusercontent.com/29004842/196052775-cd61943e-43f9-437e-aabb-8038f13bf472.png">

I checked [cusip.com](https://www.cusip.com/identifiers.html#/CUSIP) and I didn't find anything indicating that those two characters were not allowed.

**Comment by arthurdejong** [comment]
Thanks for pointing this out. It's been fixed in 1364e19

**Comment by thomkav** [comment]
A long awaited thanks @arthurdejong !
